rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -247,12 +251,63 @@ impl FmcAliasLayer {
 
         cprintln!(
             "[afmc] Image verified using Vendor ECC Key Index {}",
-            info.vendor_ecc_pub_key_idx
+            info.vendor_ecc_pub_key_idx,
         );
         report_boot_status(FmcAliasImageVerificationComplete.into());
         Ok(info)
     }
 
+    fn update_fuse_log(log_info: &ImageVerificationLogInfo) -> CaliptraResult<()> {
```

Comment:
Do we have an idea of what the fuse values will be used for by FMC/RT?

If it's just for mixing into RT Alias Key, ROM could just pass along a hash of all these fuse values.

Reply:
This is just to understand the failure of image validation. This is just for postmortem analysis of the failure. I do not think that we will be using these values for anything else.

Reply:
Ok, thanks